PackageSourceMappingExample/README.md

```diff
@@ -1,10 +1,10 @@
 # Package Namespaces example
 
-This is an example demonstrating the [package namespaces feature](https://github.com/NuGet/Home/blob/dev/proposed/2021/PackageNamespaces.md).
+This is an example demonstrating the [Package Source Mapping feature](https://github.com/NuGet/Home/blob/dev/proposed/2021/PackageNamespaces.md) (initially proposed as "Package Namespaces").
 At the time of this writing, only a limited restore functionality is supported.
 
-* [Visual Studio 2022 preview 3 and later](https://visualstudio.microsoft.com/vs/preview/).
-* [nuget.exe 6.0.0-preview.3 and later](https://www.nuget.org/downloads).
+* [Visual Studio 2022 preview 4 and later](https://visualstudio.microsoft.com/vs/preview/).
+* [nuget.exe 6.0.0-preview.4 and later](https://www.nuget.org/downloads).
 * [.NET SDK 6.0 preview 7 and later](https://devblogs.microsoft.com/dotnet/announcing-net-6-preview-7/)

```

Comment:
`.NET SDK 6.0 preview 7`  doesn't have latest bits because the latest bits because I modified code to reflect new names in the first week of Sep 2021. ``NET SDK 6.0 RC1``releasing tomorrow should have these latest bits.